Makefile

```diff
@@ -5,10 +5,10 @@ SHELL := /bin/bash
 BUILD_SUFFIX := extension
 BUILD_DIR := $(PROJECT_ROOT)/tmp/build_$(BUILD_SUFFIX)
 ZAI_BUILD_DIR := $(PROJECT_ROOT)/tmp/build_zai
+COMPONENTS_BUILD_DIR := $(PROJECT_ROOT)/tmp/build_components
 SO_FILE := $(BUILD_DIR)/modules/ddtrace.so
 WALL_FLAGS := -Wall -Wextra
-EXTRA_CFLAGS :=
-CFLAGS := -O2 $(shell [ -n "${DD_TRACE_DOCKER_DEBUG}" ] && echo -O0 -g) $(EXTRA_CFLAGS) $(WALL_FLAGS)
+CFLAGS := $(shell [ -n "${DD_TRACE_DOCKER_DEBUG}" ] && echo -O0 -g || echo -O2 -g0) $(WALL_FLAGS)
```

Comment:
@SammyK I think the `-g|-g0` was discussed previously, any memory from back then that applies to this change?

Reply:
Oh, that's a woops ... I was sure I saw g0 somewhere, but not here :)